build/dev-setup.cmd

```diff
@@ -8,136 +8,16 @@ set node-root=%~dp0..
 REM // resolve to fully qualified path
 for %%i in ("%node-root%") do set node-root=%%~fi
 
-cd %node-root%\common\core
-echo.
-echo -- Creating links for %cd% --
-call npm link
-call npm run build
-
-cd %node-root%\common\transport\amqp
-echo.
-echo -- Creating links for %cd% --
-call npm link azure-iot-common
-call npm link
-call npm run build
-
-cd %node-root%\common\transport\http
-echo.
-echo -- Creating links for %cd% --
-call npm link azure-iot-common
-call npm link
-call npm run build
-
-cd %node-root%\common\transport\mqtt
-echo.
-echo -- Creating links for %cd% --
-call npm link azure-iot-common
-call npm link
-call npm run build
-
-cd %node-root%\device\core
-echo.
-echo -- Creating links for %cd% --
-call npm link azure-iot-http-base
-call npm link azure-iot-common
-call npm link
-call npm run build
-
-cd %node-root%\service
-echo.
-echo -- Creating links for %cd% --
-call npm link azure-iot-common
-call npm link azure-iot-amqp-base
-call npm link azure-iot-http-base
-call npm link
-call npm run typings
-call npm run build
-
-cd %node-root%\device\transport\amqp
-echo.
-echo -- Creating links for %cd% --
-call npm link azure-iot-amqp-base
-call npm link azure-iot-common
-call npm link azure-iot-device
-call npm link azure-iothub
-call npm link
-call npm run build
-
-cd %node-root%\device\transport\http
-echo.
-echo -- Creating links for %cd% --
-call npm link azure-iot-http-base
-call npm link azure-iot-common
-call npm link azure-iot-device
-call npm link azure-iothub
-call npm link
-call npm run build
-
-cd %node-root%\device\transport\mqtt
-echo.
-echo -- Creating links for %cd% --
-call npm link azure-iot-mqtt-base
-call npm link azure-iot-common
-call npm link azure-iot-device
-call npm link azure-iothub
-call npm link
-call npm run build
-
-cd %node-root%\provisioning\device
-echo.
-echo -- Creating links for %cd% --
-call npm link azure-iot-common
-call npm link
-call npm run build
-
-cd %node-root%\provisioning\transport\amqp
-echo.
-echo -- Creating links for %cd% --
-call npm link azure-iot-amqp-base
-call npm link azure-iot-common
-call npm link azure-iot-provisioning-device
-call npm link
-call npm run build
-
-cd %node-root%\provisioning\transport\http
+cd %node-root%\build\tools
 echo.
-echo -- Creating links for %cd% --
-call npm link azure-iot-http-base
-call npm link azure-iot-common
-call npm link azure-iot-provisioning-device
-call npm link
-call npm run build
+echo -- Setting up links for build tools --
+call npm install
```

Comment:
>install [](start = 9, length = 7)

this will fail during the release - the reason is that the create_device.js script relies on azure-iothub and during the release the version number of the dependency is bumped before it is published.